languages/csharp/exercises/concept/dictionaries/.docs/introduction.md

```diff
@@ -0,0 +1,62 @@
+A dictionary is a collection of elements where each element comprises a key and value such that if a key is passed to a method of the ditionary its associated value is returned. It has the same role as maps or associative arrays do in other languages.
+
+A dictionary can be created as follows:
+
+```csharp
+new Dictionary<int, string>{{1, "One"}, {2, "Two"}};
```

Comment:
I was wondering if maybe having the element on a separate line would be helpful here? It also seems to be the format commonly used when initializing a dictionary (each element on a separate line).

```suggestion
new Dictionary<int, string>
{
    {1, "One"},
    {2, "Two"}
};
```